docs/csharp/quick-starts/branches-and-loops.yml

```diff
@@ -0,0 +1,241 @@
+### YamlMime:YamlDocument
+documentType: Tutorial
+title: Branches and loops
+metadata:
+  title:  Branches and loops
+  description: In this tutorial about branches and loops, you'll use your browser to learn C# interactively. You're going to write C# code and see the results of compiling and running your code directly in the browser.
+  audience: Developer
+  level: Beginner
+  displayType: two-column
+  interactive: csharp
+items:
+- durationInMinutes: 1
+  content: |
+    This tutorial teaches you about how to write code that examines variables and changes execution path based on those variables. You'll write C# interactively, using your browser to write C# and see the results of compiling and running your code. It contains a series of lessons that explore branching and looping constructs in C#. These lessons teach you the fundamentals of the C# language.
+- title: Make decisions using the if statement
+  durationInMinutes: 2
+  content: |
+    Run the following code in the interactive window. To do that, type the following code block in the interactive window and click the *Run* button:
```

Comment:
should we give a quick intro into booleans and if statement on this page? seems to go too straight to the point perhaps? could be before you introduced the first code.

Reply:
I want to 'hook' beginners with code, so I started there.

I did add a paragraph of text after running the first sample. Let me know what you think.